doc/source/developer_reference/output_processing_example.rst

```diff
@@ -1,96 +0,0 @@
-
```

Comment:
Any still relevant information from ``developer_reference/APIs.rst`` should also be moved into the new API directory and the old file deleted with its reference in ``developer_reference.rst`` removed.

Reply:
Yup, just pushed another commit that deals with that.